tests/webpack_multiple_config/package.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Would be good to auto generate the package.json too, but not very important at the moment